Microsoft.Toolkit.Uwp.UI.Controls/ColumnResizer/ColumnResizer.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+
+    <Style TargetType="local:ColumnResizer">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:ColumnResizer">
+                    <Grid  x:Name="Resizer"  
+                           Width="{TemplateBinding Width}" 
+                           Background="{TemplateBinding Background}">
+                        <TextBlock HorizontalAlignment="Center"        
```

Comment:
Perhaps better to use `GripperBarVertical` and `GripperBarHorizontal` from the MDL font, rather than 3 pipes. See the list here: https://msdn.microsoft.com/windows/uwp/style/segoe-ui-symbol-font.
The symbol should change based on whether it's row or column splitter